test/stripe/tokens_test.exs

```diff
@@ -77,7 +77,7 @@ defmodule Stripe.TokensTest do
       {:ok, res} ->
         #Apex.ap res
         assert res.id
-        assert res.status == "paid"
+        assert res.status == "succeeded"
```

Comment:
Don't want to know what a `strip dashboard` is but I'll take your word for it...